### DISCUSSION TITLE
Does Winery support x86on64?

### DISCUSSION THREAD

**Discussion by ajmas** [The-Wineskin-Project/WineskinServer#377] [Q&A] [answered]
I am on an Intel Mac and I am trying to work out if WineSkin supports x86on64, to allow running 32bit Windows software on 64-bit macOS installs?

My main issue right now is that I can't install Steam. I do have Bootcamp, but if I can avoid reboots, that would be appreciated. BTW the error I get is that I either already have the binary installed or the installer crashed. Evidence suggests the latter.

**Comment by Gcenx** [comment, marked as answer]
https://github.com/Gcenx/WineskinServer/blob/master/README.md#macos-catalina-support
https://github.com/Gcenx/WineskinServer/wiki#steam

**Reply by ajmas** [reply]
Thank you. Not sure why I couldn't find it. Steam installed and launched successfully.
